includes/gateways/paypal/includes/class-wc-gateway-paypal-ipn-handler.php

```diff
@@ -92,7 +92,7 @@ public function validate_ipn() {
 			'httpversion' => '1.1',
 			'compress'    => false,
 			'decompress'  => false,
-			'user-agent'  => 'ClassicCommerce/' . WC()->version,
+			'user-agent'  => 'WooCommerce/3.5.3 (compatible; ClassicCommerce/' . WC()->version . ')',
```

Comment:
This one is inconsistent with the rest (`ClassicCommerce/` and then the Woo version, not the CC version).

I'd suggest making a utility function like `WooCommerce::user_agent_header()` to handle all of these cases and avoid repetition/errors.  It could be as simple as this:

```php
public static function user_agent_header() {
    return sprintf( 'WooCommerce/%s (compatible; ClassicCommerce/%s)', WC()->version, WC()->cc_version );
}
```

Reply:
After taking another look I noticed this suggestion wouldn't handle the "Hookshot" user-agent strings. Maybe write the function to handle those too, or just leave them separate since there are only two?